### DISCUSSION TITLE
Note preview when hovering over card with cursor

### DISCUSSION THREAD

**Discussion by MariosGkMeng** [mgmeyers/obsidian-kanban#385] [Ideas & Feature Requests]
I am new to Kanban and I have already found it super useful!! 
So far I've organized notes in different ways. 
One suggestion for addition: When one creates note on a card, it would be very helpful to enable the feature of note preview (small window) when hovering over the card with the cursor.

**Comment by Ceasers** [comment]
you can press Ctrl(Command) at the same time when hoving to see the preview, which is an default unction by Obsidian.
